src/test/java/com/spring/assignment/employeesystem/EmployeesystemApplicationTests.java

```diff
@@ -0,0 +1,26 @@
+package com.spring.assignment.employeesystem;
+
+import com.spring.assignment.employeesystem.dao.EmployeeRepository;
+import com.spring.assignment.employeesystem.entity.Employee;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+@ContextConfiguration()
+@SpringBootTest
+@WebMvcTest
+class EmployeesystemApplicationTests {
+
+	@Test
+	public void contextLoads() {
```

Comment:
You can delete the default spring boot test class if you are not adding any tests to it